server/email/config.js

```diff
@@ -1,8 +1,8 @@
 Meteor.startup(function() {
 
   Meteor.Mailgun.config({
-    username: 'postmaster@maps.focallocal.org',
-    password: '29616944c3065bb0a0dbd3d3804ba22f'
```

Comment:
I'll remove the file from the repo and all previous commits. I think we'll have to set it up with travis as an env var.